.github/workflows/windows.yml

```diff
@@ -240,6 +256,19 @@ jobs:
           path: ${{ env.INSTALL_DIR_JS }}
           if-no-files-found: 'error'
 
+      - name: Store artifacts to a shared drive
```

Comment:
Why don't to use the same signature as github actions use (upload-artifact, restore-artifact)? then it would be easier to switch pipelines

Reply:
What exactly you mean by "switch pipelines"? I'm not going to remove the standard "upload artifacts" step